src/integration/test_client.py

```diff
@@ -205,6 +219,43 @@ def _create_responder(self):
                 .addActionFinish())
 
 
+class LetsEncryptStagingLibcloudTests(ClientTestsMixin, TestCase):
+    """
+    Tests using the real ACME client against the Let's Encrypt staging
+    environment, and the dns-01 challenge.
+
+    You must set $ACME_HOST to a hostname that will be used for the challenge,
+    and $LIBCLOUD_PROVIDER, $LIBCLOUD_USERNAME, $LIBCLOUD_PASSWORD, and
+    $LIBCLOUD_ZONE to the appropriate values for the DNS provider to complete
+    the challenge with.
+    """
+    HOST = _getenv(u'ACME_HOST')
+    PROVIDER = _getenv(u'LIBCLOUD_PROVIDER')
```

Comment:
There's a potential namespacing issue here.  In my opinion, all variables that influence the operation of txacme should be prefixed `TXACME_`.  This sort of cross-library bleed-over is how we got httpoxy, after all.